golem-worker-executor-base/src/services/compiled_template.rs

```diff
@@ -47,6 +48,7 @@ pub async fn configured(
                 aws_config::defaults(BehaviorVersion::v2023_11_09()).region(Region::new(region));
 
             let sdk_config = if let Some(endpoint_url) = &config.aws_endpoint_url {
+                info!("The endpoint urls is {}", &endpoint_url);
```

Comment:
This didn't work at all , while I was able to login to worker container and query the s3 buckets in localstack usintg the endpoint URL

The issue is

```scala
timestamp":"2024-01-18T07:57:55.896743Z","level":"WARN","message":"Failed to upload compiled template TemplateKey { template_id: TemplateId(cc2cffa9-57d5-4165-a452-0108c954345b), template_version: 0 }: Failed to download template: cc2cffa9-57d5-4165-a452-0108c954345b#0: Could not upload compiled template: dispatch failure: io error: error trying to connect: dns error: failed to lookup address information: Name or service not known: dns error: failed to lookup address information: Name or service not known: failed to lookup address information: Name or service not known (DispatchFailure(DispatchFailure { source: ConnectorError { kind: Io, source: hyper::Error(Connect, ConnectError(\"dns error\", Custom { kind: Uncategorized, error: \"failed to lookup address information: Name or service not known\" })), connection: Unknown } }))","target":"golem_worker_executor_base::services::template"}
```

while I was able to do

````
Unpacking less (590-2) ...
Setting up less (590-2) ...
Processing triggers for mailcap (3.70+nmu1) ...
# aws s3 ls --endpoint-url http://localstack:4566
2024-01-18 05:18:44 golem-compiled-components
# %
 ✘ afsalthaj@Afsals-MacBook-Pro  ~/projects/golem-services   persistent_volume ±  docker ps
CONTAINER ID   IMAGE                                              COMMAND                  CREATED         STATUS                   PORTS                                                                                                                             NAMES
```